common/src/main/java/org/apache/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -86,9 +112,20 @@ public String toString()
            ", connectURI='" + getConnectURI() + '\'' +
            ", user='" + user + '\'' +
            ", passwordProvider=" + passwordProvider +
+           ", dbcpPropertiesFile=" + dbcpPropertiesFile +
            '}';
   }
 
+  String getDbcpPropertiesFile()
+  {
+    return dbcpPropertiesFile;
+  }
+
+  void setDbcpPropertiesFile(String dbcpPropertiesFile)
```

Comment:
Can we avoid this setter method by creating a Properties object within the `TestDerbyConnector` class and this object could be created off of a temporary properties file within the test class?

Reply:
yep, let me change that.